test/Stats.unittest.js

```diff
@@ -135,24 +167,18 @@ describe(
 					},
 					compiler: {
 						context: ""
+					},
+					hooks: {
+						stats: new SyncHook(["stats"])
```

Comment:
Let's keep mocking it

Reply:
We need hooks, as compilation is required to have hooks property now, as stats constructor will call stats hook on compilation, should, I implement a mock hook call?